component/module/file/file.go

```diff
@@ -14,8 +14,9 @@ import (
 
 func init() {
 	component.Register(component.Registration{
-		Name:    "module.file",
-		Args:    Arguments{},
+		Name: "module.file",
+		Args: Arguments{},
+		//nolint:staticcheck
 		Exports: module.Exports{},
```

Comment:
nit: I find the nolint comments visually distracting from the reading flow. Does it still work if we have the comment inline like this?

```go
Exports: module.Exports{}, //nolint:staticcheck
```

ditto for anywhere else we added that comment.